docs/Module - Github.md

```diff
@@ -0,0 +1,89 @@
+## Module - GitHub
+ File : [`.github/workflows/module-github.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-github.yaml)
+
+### Purpose
+The `module-manage-github` workflow serves as an entry point for GitHub management tasks. It calls modular workflows for managing GitHub projects, handling pull requests, and automating releases.
+
+### Triggers
+This workflow is triggered using `workflow_call`, allowing other workflows to invoke it with specific inputs.
+
+### Inputs 
+-  `run_job_project` :  
+  Description: Determines whether to run the project management job  
+  Type: `boolean`  
+  Default: `true`
+
+### Jobs 
+1.  Project Management :
+    - Executes the [`module-github-project.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-github.yaml) workflow
+    - Runs when:
+        - `run_job_project` is `true`, and
+        - The event is a pull request or issue
+2.  Pull Request Management :
+    - Executes the [`module-github-pull-request.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-github-pull-request.yaml) workflow
+    - Runs for pull request events
+3.  Release Automation :
+    - Executes the [`module-github-release.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-github-release.yaml) workflow
+    - Runs when:
+        - The event is a push to the `main` branch
```

Comment:
This file also contains far too much detail. It’s essentially the workflow written in Markdown instead of YAML. The same issue applies: if we make even small changes to the workflow (YAML), we’ll need to update this documentation as well. This creates unnecessary overhead that we should avoid.